packages/hardhat-test-helpers/src/helpers/impersonateAccount.ts

```diff
@@ -0,0 +1,12 @@
+import { getHardhatProvider, assertValidAddress } from "../utils";
+
+export async function impersonateAccount(hexAddress: string): Promise<void> {
```

Comment:
Adding this comment just as a reminder that we need to add jsdocs to all the public functions.